node/src/actors/chain_manager/handlers.rs

```diff
@@ -1001,7 +1001,7 @@ impl Handler<PeersBeacons> for ChainManager {
                             // machine to move into `Synced` state.
                             log::debug!("Moving from AlmostSynced to Synced state");
                             log::info!("{}", SYNCED_BANNER);
```

Comment:
should we add the banner inside the function?

Reply:
Done!